releases/katello/4.14/settings

```diff
@@ -0,0 +1,4 @@
+FULLVERSION='4.14.0-rc1'
+FOREMAN_VERSION='3.12'
+OSES="el8 el9"
+KOJI_TAGS="katello-@VERSION@-@OS@ katello-candlepin-@VERSION@-el8"
```

Comment:
This isn't needed anymore so I've opened https://github.com/theforeman/theforeman-rel-eng/pull/395